presto-main/src/test/java/io/prestosql/tests/ReportUnannotatedMethods.java

```diff
@@ -92,6 +92,10 @@ private static boolean isTestAnnotated(Method method)
                         // testng annotation (@Test, @Before*, @DataProvider, etc.)
                         return true;
                     }
+                    if ("io.prestosql.tempto".equals(annotationClass.getPackage().getName())) {
```

Comment:
maybe let's use starts with `io.prestosql.tempto`?

Reply:
it would match `io.prestosql.temptor`, which is not a sub-package, and supporting `=tempto  OR startswith io.prestosql.tempto.` is too much writing.